fontique/src/backend/coretext.rs

```diff
@@ -37,7 +41,17 @@ pub struct SystemFonts {
 
 impl SystemFonts {
     pub fn new() -> Self {
-        let scanned = scan::ScannedCollection::from_paths(Some("/System/Library/Fonts"), 8);
+        let paths = unsafe {
+            NSSearchPathForDirectoriesInDomains(
+                NSSearchPathDirectory::NSLibraryDirectory,
+                NSSearchPathDomainMask::NSAllDomainsMask,
+                Bool::YES,
+            )
+            .as_ref()
+            .iter()
+            .map(|p| p.to_string() + "/Fonts/")
```

Comment:
This looks like a potential double-allocation hazard - if the path is nearly a power of two length, this will allocate a second time (at least according to https://github.com/rust-lang/rust/issues/16415#issuecomment-65233945).

I that this should be:
```suggestion
            .map(|p| format!("{p}/Fonts/"))
```

Reply:
Good call! Done and updated.